utils/colors.ts

```diff
@@ -1,29 +0,0 @@
-/**
```

Comment:
What happened to this?

Reply:
Moved into StyleSheet.ts

Reply:
Whoops. That was lost in translation. Adding comments